wrapper/lib/zn-data.js

```diff
@@ -173,10 +173,8 @@ export function ZnData (plugin) {
           }
         }
 
-        // Set wheter it's a bulk or single operation
-        if (angular.isArray(data) || isMultiIdField || hasBatchConditions) {
-
-        } else if (method === 'put' && typeof params[idField] === 'undefined') {
+        // Set whether it's a bulk or single operation
+        if (!(angular.isArray(data) || isMultiIdField || hasBatchConditions) && method === 'put' && typeof params[idField] === 'undefined') {
```

Comment:
Not sure why that block was empty, so if it should have had some logic in it, then this can be fixed. Otherwise, this is the same logic in a single if condition.